Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("", Name ="Home", Order = 0)]
+		[Route("app", Name = "app", Order = 0)]

```

Comment:
I wonder if we should make the `Order` of the `app` route `1` just for consistency.  At the moment it has the same `Order` as the base route.  I suspect no problems will be caused with shared order, but I think being explicit will help make it more clear to the reader which routes should be evaluated first.